<launch>

  <arg name="ids"                     doc="Specify the IDs of the dynamixels that are hooked up, separated by commas. Order matters."/>

  <arg name="device_name"             default="/dev/ttyUSB0"/>
  <arg name="baud_rate"               default="57600"/>

  <arg name="profile_velocity"        default="200"/>
  <arg name="profile_acceleration"    default="50"/>

  <param name="device_name"           value="$(arg device_name)"/>
  <param name="baud_rate"             value="$(arg baud_rate)"/>

  <param name="profile_velocity"      value="$(arg profile_velocity)"/>
  <param name="profile_acceleration"  value="$(arg profile_acceleration)"/>

  <param name="id_list"               value="$(arg ids)"/>

  <node name="general_velocity_control"
        pkg="dynamixel_workbench_controllers"
        type="general_velocity_control"
        required="true"
        output="screen">
  </node>

</launch>
